mockApis/interventionsService.ts

```diff
@@ -0,0 +1,1007 @@
+import Wiremock from './wiremock'
+
+export default class InterventionsServiceMocks {
```

Comment:
I assume this was just copied temporarily to get things working or as an example for when we need to mock things later.